`runtime error: index out of range` on `--waste`

Getting the following on `--waste` (with our without region specification)

```
 ~/go/bin/aws-doctor --region us-east-1 --waste
      ___           ___           ___            _____          ___           ___                       ___           ___
     /  /\         /__/\         /  /\          /  /::\        /  /\         /  /\          ___        /  /\         /  /\
    /  /::\       _\_ \:\       /  /:/_        /  /:/\:\      /  /::\       /  /:/         /  /\      /  /::\       /  /::\
   /  /:/\:\     /__/\ \:\     /  /:/ /\      /  /:/  \:\    /  /:/\:\     /  /:/         /  /:/     /  /:/\:\     /  /:/\:\
  /  /:/~/::\   _\_ \:\ \:\   /  /:/ /::\    /__/:/ \__\:|  /  /:/  \:\   /  /:/  ___    /  /:/     /  /:/  \:\   /  /:/~/:/
 /__/:/ /:/\:\ /__/\ \:\ \:\ /__/:/ /:/\:\   \  \:\ /  /:/ /__/:/ \__\:\ /__/:/  /  /\  /  /::\    /__/:/ \__\:\ /__/:/ /:/___
 \  \:\/:/__\/ \  \:\ \:\/:/ \  \:\/:/~/:/    \  \:\  /:/  \  \:\ /  /:/ \  \:\ /  /:/ /__/:/\:\   \  \:\ /  /:/ \  \:\/:::::/
  \  \::/       \  \:\ \::/   \  \::/ /:/      \  \:\/:/    \  \:\  /:/   \  \:\  /:/  \__\/  \:\   \  \:\  /:/   \  \::/~~~~
   \  \:\        \  \:\/:/     \__\/ /:/        \  \::/      \  \:\/:/     \  \:\/:/        \  \:\   \  \:\/:/     \  \:\
    \  \:\        \  \::/        /__/:/          \__\/        \  \::/       \  \::/          \__\/    \  \::/       \  \:\
     \__\/         \__\/         \__\/                         \__\/         \__\/                     \__\/         \__\/

 🏥 AWS DOCTOR CHECKUP
 Account ID: xxxxxxxxxxx
 ------------------------------------------------
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/elC0mpa/aws-doctor/utils.drawEBSTable({0x102cf6b40, 0x0, 0x0}, {0x140001241e0, 0x2, 0x2})
	~/go/pkg/mod/github.com/el!c0mpa/aws-doctor@v1.0.0/utils/waste_table.go:63 +0x548
github.com/elC0mpa/aws-doctor/utils.DrawWasteTable({0x14000113260, 0xc}, {0x0, 0x0, 0x0}, {0x102cf6b40, 0x0, 0x0}, {0x140001241e0, 0x2, ...}, ...)
	~/go/pkg/mod/github.com/el!c0mpa/aws-doctor@v1.0.0/utils/waste_table.go:32 +0x200
github.com/elC0mpa/aws-doctor/service/orchestrator.(*service).wasteWorkflow(0x14000279c50)
	~/go/pkg/mod/github.com/el!c0mpa/aws-doctor@v1.0.0/service/orchestrator/service.go:111 +0x18c
github.com/elC0mpa/aws-doctor/service/orchestrator.(*service).Orchestrate(0x16da57514?, {{0x16da57514, 0x9}, {0x0, 0x0}, 0x0, 0x1})
	~/go/pkg/mod/github.com/el!c0mpa/aws-doctor@v1.0.0/service/orchestrator/service.go:23 +0x54
main.main()
	~/go/pkg/mod/github.com/el!c0mpa/aws-doctor@v1.0.0/app.go:37 +0x2b4
```

--- a/utils/waste_table.go
+++ b/utils/waste_table.go
@@ -27,7 +27,6 @@ func DrawWasteTable(accountId string, elasticIpInfo []types.Address, unusedEBSVo
 		return
 	}
 
-	// --- Existing Logic ---
 	if len(unusedEBSVolumeInfo) > 0 || len(attachedToStoppedInstancesEBSVolumeInfo) > 0 {
 		drawEBSTable(unusedEBSVolumeInfo, attachedToStoppedInstancesEBSVolumeInfo)
 	}
@@ -56,27 +55,29 @@ func drawEBSTable(unusedEBSVolumeInfo []types.Volume, attachedToStoppedInstances
 		},
 	})
 
-	statusAvailable := "Available (Unattached)"
-	rows := populateEBSRows(unusedEBSVolumeInfo)
+	if len(unusedEBSVolumeInfo) > 0 {
+		statusAvailable := "Available (Unattached)"
+		rows := populateEBSRows(unusedEBSVolumeInfo)
 
-	halfRow := len(rows) / 2
-	rows[halfRow][0] = text.FgHiRed.Sprint(statusAvailable)
+		halfRow := len(rows) / 2
+		rows[halfRow][0] = text.FgHiRed.Sprint(statusAvailable)
 
-	t.AppendRows(rows)
-
-	rows = []table.Row{}
+		t.AppendRows(rows)
+	}
 
 	if len(unusedEBSVolumeInfo) > 0 && len(attachedToStoppedInstancesEBSVolumeInfo) > 0 {
 		t.AppendSeparator()
 	}
 
-	statusStopped := "Attached to Stopped Instance"
-	rows = populateEBSRows(attachedToStoppedInstancesEBSVolumeInfo)
+	if len(attachedToStoppedInstancesEBSVolumeInfo) > 0 {
+		statusStopped := "Attached to Stopped Instance"
+		rows := populateEBSRows(attachedToStoppedInstancesEBSVolumeInfo)
 
-	halfRow = len(rows) / 2
-	rows[halfRow][0] = text.FgHiRed.Sprint(statusStopped)
+		halfRow := len(rows) / 2
+		rows[halfRow][0] = text.FgHiRed.Sprint(statusStopped)
 
-	t.AppendRows(rows)
+		t.AppendRows(rows)
+	}
 
 	if t.Length() > 0 {
 		t.Render()
@@ -102,7 +103,6 @@ func drawEC2Table(instances []types.Instance, ris []model.RiExpirationInfo) {
 		statusLabel := "Stopped Instance(> 30 Days)"
 		rows := populateInstanceRows(instances)
 
-		// Apply grouped status label
 		halfRow := len(rows) / 2
 		rows[halfRow][0] = text.FgHiRed.Sprint(statusLabel)
 
@@ -110,9 +110,7 @@ func drawEC2Table(instances []types.Instance, ris []model.RiExpirationInfo) {
 		hasPreviousRows = true
 	}
 
-	// --- SECTION 2 & 3: Reserved Instances ---
 	if len(ris) > 0 {
-		// We split RIs into two groups for better status labeling
 		var expiring, expired []model.RiExpirationInfo
 		for _, ri := range ris {
 			if ri.Status == "EXPIRING SOON" {
@@ -122,7 +120,6 @@ func drawEC2Table(instances []types.Instance, ris []model.RiExpirationInfo) {
 			}
 		}
 
-		// Group: Expiring Soon
 		if len(expiring) > 0 {
 			if hasPreviousRows {
 				t.AppendSeparator()
@@ -137,7 +134,6 @@ func drawEC2Table(instances []types.Instance, ris []model.RiExpirationInfo) {
 			hasPreviousRows = true
 		}
 
-		// Group: Recently Expired
 		if len(expired) > 0 {
 			if hasPreviousRows {
 				t.AppendSeparator()

